pom.xml

```
@@ -49,7 +48,7 @@
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
         <artifactId>bom-${jenkins.baseline}.x</artifactId>
-        <version>3435.v238d66a_043fb_</version>
+        <version>3867.v17b_41ce6fcb_c</version>
```

Need to recommend a released version, not an incremental
```suggestion
        <version>3850.vb_c5319efa_e29</version>
```

---

https://github.com/jenkins-infra/plugin-modernizer-tool/issues/534